test/redis-sentinel/log/.gitignore

```diff
@@ -0,0 +1 @@
+./*.log
```

Comment:
Could all these .gitignores be merged into a single one in the root dir?

Reply:
hm ... it is the question ... splitting is more special and so you know why things are where ...